From d6b0efcdd83a937363267ed751fd9a1085505394 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 20 Apr 2021 17:06:21 +0200
Subject: [PATCH] sd-device: use right type for usec_initialized

(cherry picked from commit a156eb89c827206ee5b51d53016ba63be0c90449)

Reference: https://github.com/systemd/systemd-stable/commit/d6b0efcdd83a937363267ed751fd9a1085505394
Conflict: NA
---
 src/libsystemd/sd-device/device-internal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libsystemd/sd-device/device-internal.h b/src/libsystemd/sd-device/device-internal.h
index c1a81e3b41..c7f9b59087 100644
--- a/src/libsystemd/sd-device/device-internal.h
+++ b/src/libsystemd/sd-device/device-internal.h
@@ -69,7 +69,7 @@ struct sd_device {
 
         char *id_filename;
 
-        uint64_t usec_initialized;
+        usec_t usec_initialized;
 
         mode_t devmode;
         uid_t devuid;
-- 
2.27.0

